src/FantomasOptionsPage2.xaml

```diff
@@ -0,0 +1,40 @@
+﻿<UserControl
```

Comment:
I imagine the page to be some kind of fusion of

Resharper settings:
![image](https://user-images.githubusercontent.com/11340282/103655091-0d66d180-4f67-11eb-8ed3-ba1636b0ea59.png)

The old C# settings:
![image](https://user-images.githubusercontent.com/11340282/103655160-28394600-4f67-11eb-8fff-1eae74c23159.png)

The new C# settings:
![image](https://user-images.githubusercontent.com/11340282/103655227-3c7d4300-4f67-11eb-8806-45ea2c935d0d.png)